lib/gli/option_handler_patch.rb

```diff
@@ -0,0 +1,21 @@
+# FIXME:
+GLI::OptionParserFactory.class_eval do
+  def self.setup_options(opts, tokens, options)
+    tokens.each do |ignore, token|
+      opts.on *token.arguments_for_option_parser do |arg|
+        token.names_and_aliases.each do |name|
+          if token.respond_to? :handler then token.handler arg
```

Comment:
This is a super ugly monkey patch to support basic option handling,
but there must be a proper way. Only this line is new, the rest is a
copy of original method, we cannot use such a patch.